### ISSUE TITLE
Rename jinahub folder?

### ISSUE THREAD

**Issue by tadejsv** [jina-ai/executors#66]
Since we are only have all the executors here as a convenience for us, shouldn't we rename the folder from `jinahub` to something else? Seems this way people might confuse this repo with the actual hub.

**Comment by tadejsv** [comment]
No, afaik it's there only for some relative imports, right @cristianmtr ?

**Comment by cristianmtr** [comment]
Yes, not sure if importing in core relies on the name. Needs to be tested if we rename it

**Comment by tadejsv** [comment]
The core importing does not rely on it. It only cares about the name of the file it is importing.
In fact, because core can actually prepend `jinahub` to a module name if it conflicts with something already imported, I think it is best that we rename it to something else
